packages/core/src/utils.js

```diff
@@ -1048,6 +1048,36 @@ export function toDateString(
   return time ? datetime : datetime.slice(0, 10);
 }
 
+export function utcToLocal(jsonDate) {
```

Comment:
What exactly is the point of this function?

Reply:
This is what ensures the format is correct in the Bootstrap theme. I didn’t write this, just simply relocated it to utils.